00_Base/src/util/util.ts

```diff
@@ -15,12 +15,14 @@ export enum CountryCode {
 }
 
 export const plainToClass = <T>(constructor: Constructable<T>, plain: T): T =>
```

Comment:
it is possible to pass additional options to `plainToInstance` and `instanceToPlain` methods so centralizing via a helper method like this can help us apply such a config across all usages - otherwise you are correct, it isnt needed, and  currently we don't pass in a config, as we previously did, but I think it is still good just in case we do face the need since we already have it...